runtime/oti/j9nonbuilder.h

```diff
@@ -2268,6 +2268,9 @@ typedef struct J9ROMMethodHandleRef {
 #define MN_IS_TYPE			0x00080000
 #define MN_CALLER_SENSITIVE	0x00100000
 #define MN_TRUSTED_FINAL	0x00200000
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#define MN_FLATTENED		0x00400000
+#endif /* #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
```

Comment:
nit: Please omit the `#if` part of this comment; it's not recommended by the [coding standard](https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md).

Reply:
updated in 8533190